src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.SkeletonReferenceCache.cs

```diff
@@ -57,10 +56,18 @@ internal partial class SolutionCompilationState
     /// memory.
     /// </para>
     /// </summary>
-    private partial class SkeletonReferenceCache
+    [NonCopyable]
```

Comment:
prevents accidental copies, which could produce torn views of the struct.  Clone is the only supported mechanism to get another copy.  will doc.